src/aof.c

```diff
@@ -290,6 +293,15 @@ int startAppendOnly(void) {
     server.aof_last_fsync = server.unixtime;
     server.aof_fd = newfd;
 
+    /* If AOF fsync error in bio job, we just ignore it and log the event. */
+    int aof_bio_fsync_status;
+    atomicGet(server.aof_bio_fsync_status, aof_bio_fsync_status);
+    if (aof_bio_fsync_status == C_ERR) {
+        serverLog(LL_WARNING,
+            "AOF reopen, just ignore the AOF fsync error in bio job");
+        atomicSet(server.aof_bio_fsync_status,C_OK);
+    }
+
```

Comment:
why do we want to log the last status? we're gonna start with a clean one anyway?

Reply:
Jut like the flowing `server.aof_last_write_status`, actually i am not sure

Reply:
Just like the following `aof_last_write_status`, i am not sure we need

Reply:
well, i don't see why either of them is needed, but ok, no harm in one more log message on very rare cases.